wpilibc/src/main/native/include/IterativeRobotBase.h

```diff
@@ -40,7 +41,7 @@ namespace frc {
  *   - TeleopPeriodic()
  *   - TestPeriodic()
  */
-class IterativeRobotBase : public RobotBase {
+class IterativeRobotBase : public RobotBase, public IterativeBase {
  public:
   virtual void RobotInit();
```

Comment:
If an interface is being implemented, the comment on this class describing the functions should be omitted and the `override` keyword should be added. Same with Java but with the `@Override` attribute.